maestrowf/maestro.py

```diff
@@ -61,17 +62,43 @@
 
 def status_study(args):
     """Check and print the status of an executing study."""
-    status = Conductor.get_status(args.directory)
-
-    if status:
-        print(tabulate.tabulate(status, headers="keys"))
-        return 0
+    #args_dir is the path specified
+    args_dir = str(args.directory)
+    args_dir = args_dir.replace("[", "")
+    args_dir = args_dir.replace("]", "")
+    args_dir = args_dir.replace("'", "")
+
+    #Get a list of all the directories based on the path
+    directory_list = []
+    directory_list = args_dir.split(',')
+
+    dir_len = len(directory_list)
+
+    if dir_len == 1:
```

Comment:
You don't need to differentiate between a length of 1 and multiple. The only thing you probably need to catch is if the length of the list of directories is 0. That means no valid path was provided (though that might get resolved by the argument parser due to the `nargs` specifying that at least one argument must be specified).